release-notes.hbs.md

```diff
@@ -144,6 +144,18 @@ enable TUF:
 
 - SCST - Store returns an expired certificate error message when a CA certificate expires before the app certificate. For more information, see [CA Cert expires](scst-store/troubleshooting.hbs.md#ca-cert-expires).
 
+#### <a id='1-9-0-ssc-ui-ki'></a> v1.9.0 Known issues: Supply Chain UI
+
+- When accessing the supply chain tab in the Tanzu Developer Portal, users might encounter an error related to data.packaging.carvel.dev. This issue arises even for supply chains that do not generate Carvel packages, indicating an unexpected configuration requirement. The error message displayed is related to permission issues and JSON parsing errors, specifically mentioning that the user "system:serviceaccount:tap-gui:tap-gui-viewer" cannot list resource "packages" in the API group "data.packaging.carvel.dev" at the cluster scope. Additionally, an unexpected non-whitespace character is reported after JSON at position 4.
```

Comment:
**SUGGESTION**

Flip this sentence: "This issue indicates a misconfiguration and arises even for supply chains that do not generate Carvel packages."

Reply:
Removed!